tutoroars/templates/oars/apps/pythonpath/superset-dashboard.py

```diff
@@ -52,15 +73,25 @@ def update_assets():
     dashboard.save()
 
 
-def zip_dir(zip_dir):
+def superset_asset_zip(zip_dir, metadata_type, metadata_version='1.0.0'):
     """
     Zips up the contents of the given dir to a temporary file,
-    and returns the temporary file pointer.
+    adding in the expected metadata.yaml file for the given asset type.
+
+    Returns the temporary file pointer to the zip file.
     """
     fp = tempfile.NamedTemporaryFile(delete=False, suffix=".zip")
     asset_dir = os.path.abspath(zip_dir)
     archive_base_dir = 'assets'
+    timestamp = datetime.datetime.utcnow().replace(tzinfo=datetime.timezone.utc).isoformat()
     with zipfile.ZipFile(fp, "w") as ziph:  # ziph is zipfile handle
+        # Write a metadata.yaml file to the root dir
+        ziph.writestr(os.path.join(archive_base_dir, 'metadata.yaml'),
```

Comment:
@bmtcril Ah.. I think that's because there's old OARS assets files still in your tutor config dirs. I've removed those `metadata.yaml` files here and am creating them on demand instead, so that I can import (and overwrite) each asset type individually. (I learned that Superset is fussy about both the `metadata.yaml` file contents and the base dir of the zip file when importing individual assets.)

Have updated the PR test instructions here.. could you try that with my latest code?